.github/ISSUE_TEMPLATE/scraper_bug_report.yml

```diff
@@ -0,0 +1,83 @@
+name: Apify scraper bug report
+description: Report incorrect or unexpected behavior of a scraper
+labels: [bug]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        If you have a question, please ask it on [GitHub Discussions](https://github.com/apify/crawlee/discussions) or in our [Discord server](https://discord.gg/jyEM2PRvMU)
+
+  - type: dropdown
+    id: scraper
+    attributes:
+      label: Which scraper is this bug report for?
+      multiple: false
+      options:
+        - "cheerio-scraper"
+        - "playwright-scraper"
+        - "puppeteer-scraper"
+        - "web-scraper"
+    validations:
+      required: true
+
+  - type: textarea
+    id: description
+    attributes:
+      label: Issue description
+      description: |
+        Describe the issue in as much detail as possible.
+
+        Tip: You can attach images or log files by clicking this area to highlight it and then dragging files into it.
+      placeholder: |
+        Steps to reproduce with the code sample below:
+        1. do thing
+        2. call x method
+        3. observe behavior
+        4. see error logs below
+    validations:
+      required: true
+
+  - type: textarea
+    id: codesample
+    attributes:
+      label: Code sample
+      description: Include a reproducible, minimal code sample. This will be automatically formatted into code, so no need for backticks.
+      render: TypeScript
+      placeholder: |
+        // Your code sample should be...
+        // ... Minimal - use as little code as possible that still produces the same problem  (and is understandable)
+        // ... Complete - Provide all parts someone else needs to reproduce your problem
+        // ... Reproducible - Test the code you're about to provide to make sure it reproduces the problem
+
+  - type: input
+    id: scraper-version
+    attributes:
+      label: Scraper version
+      description: Which version of the scraper are you using? Check it on the Apify console and paste the output.
+    validations:
+      required: true
+
+  - type: input
+    id: actor
+    attributes:
+      label: Run id
+      description: If you have the run id that shows this issue, feel free to share it here, as it can help us debug it further
+
+  - type: dropdown
+    id: priority
+    attributes:
+      label: Priority this issue should have
+      description: Make sure you choose something that makes sense! If you need to elaborate on your reasoning, please do so in the Issue description section.
+      multiple: false
+      options:
+        - "Low (slightly annoying)"
+        - "Medium (should be fixed soon)"
+        - "High (immediate attention needed)"
+    validations:
+      required: true
```

Comment:
i would drop this, everything has high priority for user that has the problem :]